modules/apps/batch-engine/batch-engine-test/src/testFunctional/tests/ProcessBatchEngineFilesFoundInDXPModules.testcase

```diff
@@ -0,0 +1,232 @@
+@component-name = "portal-batch-engine"
+definition {
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Batch Engine";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		BatchEngine.setUpGlobalIds();
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if (${testPortalInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			BlogPostingAPI.deleteAllBlogPostings();
+
+			ObjectAdmin.deleteAllCustomObjectsViaAPI();
+		}
+	}
+
+	@priority = 4
+	test CanProcessJarWithBlogBatchEngineFiles {
+		property portal.acceptance = "true";
+
+		task ("Given an osgi jar file with blogPosting batch engine file and Liferay-Client-Extension-Batch header") {
+			var projectDir = PropsUtil.get("project.dir");
```

Comment:
Shall we use the existing copyDependenciesDirToFolder instead of making destinationDir so long?